styles/globals.css

```diff
@@ -58,6 +58,6 @@ body {
   }
 
   .transaction-item {
-    @apply flex justify-between border-b-2 mb-4 pb-4;
+    @apply flex justify-between border-b-2 mb-4 pb-4 last:pb-0 last:mb-0 last:border-none;
```

Comment:
ah @gglucass I was confused what that was doing, I independently moved the eval logic to the component style because I thought it would be more semantic.